server/test/uk/gov/ons/addressIndex/server/modules/ElasticsearchRepositorySpec.scala

```diff
@@ -427,34 +443,42 @@ class ElasticsearchRepositorySpec extends WordSpec with SearchMatchers with Clas
     "lpi" -> Seq(),
     "paf" -> Seq(fourthHybridPafEs))
 
-  testClient.execute {
-    createIndex(hybridIndexName)
-      .mappings(MappingDefinition.apply(hybridMappings))
-      .analysis(Some(CustomAnalyzerDefinition("welsh_split_synonyms_analyzer",
-        StandardTokenizer("myTokenizer1"))
-      ))
-  }.await
+  // new analysis object, doesn't seem to work
+  val customAnalyzer = CustomAnalyzer ("welsh_split_synonyms_analyzer","myTokenizer1",List(),List())
+  val testAnalysis: Analysis = Analysis(
+    List(customAnalyzer))
 
+  // todo get it to work with new analysis package
   testClient.execute {
-    createIndex(hybridIndexHistoricalName)
-      .mappings(MappingDefinition.apply(hybridMappings))
+    createIndex(hybridIndexName)
+//      .analysis(testAnalysis)
```

Comment:
This comment probably isn't needed

Reply:
This one is. The tests fail using the new analysis package instead of the old one (don't know why) so it is commented out for now.